### DISCUSSION TITLE
Metadata not loaded for ...

### DISCUSSION THREAD

**Discussion by TheZoker** [immich-app/immich#4203] [Q&A]
Hi there,

Yesterday I setup immich. I use a external library provided by my synology NAS. Most of the stuff worked without any issues. 
But some images are displayed in a random place. 
When I click on the image, immich shows me a:
```
Metadata not loaded for [image_path]
```

I then run the `EXTRACT METADATA` job again for missing items. Did not solve the issue.
I then run the `EXTRACT METADATA` job for all items, but still does not fix the issue. 

Most images show the meta correctly, but some do not work. 

How can I fix that? 

Thanks!

**Comment by madkatz01** [comment]
I have this issue as well - but wasn't present on Immich releases before 1.79. Might be related to the exiftool changes and seems related to this other post #4166

**Reply by TheZoker** [reply]
Alright, thanks for the hint! Good to know it's not a misconfiguration or something

**Comment by alextran1502** [comment]
Hello, is it a file that you can share for troubleshooting?

**Reply by madkatz01** [reply]
Here is one that it can't extract metadata for anymore:
![Israel 089](https://github.com/immich-app/immich/assets/82177105/ea1de13b-3f59-421e-bfa2-938bd9b56871)

**Reply by TheZoker** [reply]
Here is another one:
![IMG_2569](https://github.com/immich-app/immich/assets/1368405/a94d1bc6-ee46-483c-a84c-e6dde7ca2b5a)

**Comment by nikolajhendel** [comment]
Just to add to this.
I installed Immich 5 days ago (docker) using external library functionality as well. Index went mostly well (25,000 assets = 130GB) - but I have same issues as reported above across a subset of images.

Apart from "Metadata not loaded" issue, it also seems like the UI selects a random date to show the images in the timeline.

**Comment by alextran1502** [comment]
This will be fixed in the next release

**Reply by TheZoker** [reply]
FYI: I just updated the microservice to the latest `main` tag (instead of release) and it fixed the issue 🎉
